lib/actions/install.js

```diff
@@ -103,26 +103,34 @@ install.installDependencies = function (options) {
 
   options = _.defaults(options || {}, {
     bower: true,
-    npm: true,
+    npm: false,
     skipMessage: false,
     callback: function () {}
   });
 
+
   if (options.npm) {
```

Comment:
Yes, I understand you, thanks 👍 

My question now, Is correct the new code? I'm not sure if is the best practise, set for default npm to false.